project/glite-ce-blahp_sl_any.spec

```diff
@@ -82,7 +82,6 @@ fi
 %defattr(-,root,root)
 %config(noreplace) /etc/blparser.conf.template
 %config(noreplace) /etc/blah.config.template
-%dir /etc/rc.d/init.d/
```

Comment:
On a Scientific Linux release 6.6 I have:
rpm -qif /etc/rc.d/init.d/ | grep ^Name
Name        : chkconfig                    Relocations: (not relocatable)
Name        : glite-ce-blahp               Relocations: (not relocatable)
Name        : initscripts                  Relocations: (not relocatable)
The package initscripts is always installed so the change should not cause any problem